dts/bindings/bluetooth/zephyr,bt-hci-virtual.yaml

```diff
@@ -0,0 +1,11 @@
+description: Bluetooth HCI provided over a virtual bus
+
+compatible: "zephyr,bt-hci-virtual"
```

Comment:
There are other HCI drivers providing access to a "local" controller, typically implemented as a binary blob (west blobs fetch ...), but still the same architecture as with the native controller or the SoftDevice one:
```shell
❯ git grep VIRTUAL dts/bindings/bluetooth/
dts/bindings/bluetooth/bt-hci.yaml:      - "BT_HCI_BUS_VIRTUAL"
dts/bindings/bluetooth/infineon,cat1-bless-hci.yaml:    default: "BT_HCI_BUS_VIRTUAL"
dts/bindings/bluetooth/silabs,bt-hci.yaml:    default: "BT_HCI_BUS_VIRTUAL"
dts/bindings/bluetooth/zephyr,bt-hci-ll-sw-split.yaml:    default: "BT_HCI_BUS_VIRTUAL"
```

In order not to cause confusion, I don't think we should have one of these bindings call itself generically "bt-hci-virtual". When using the SoftDevice controller, don't you anyway need a different set of Kconfig options enabled? If so, then having a dedicated binding for it would make sense as that could be linked to a separate KConfig option.

Reply:
I suggested the unique binding name, as the DT Kconfig option would select the corresponding Controller Kconfigs. 

> What we want to avoid is to overrides the device tree bindings in the zephyr repository or require adding an additional device tree property to all the socs.

Some of the SoCs/broads/applications would switch Controller depending on feature/code size, it is best to have separate binding name. (i see it like switching BT UART, console UART etc.)

Reply:
> require adding an additional device tree property to all the socs

An option is to add a SDC dts overlay to all radio cores in NCS. In that one, we would override the `zephyr,bt-ll-sw-split` prop. 
I prefer to keep a one-to-one relationship between `compatible` string and `.c` driver source if possible.